alicloud/data_source_alicloud_cen_instances.go

```diff
@@ -0,0 +1,236 @@
+package alicloud
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/cbn"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceAlicloudCenInstances() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAlicloudCenInstancesRead,
+
+		Schema: map[string]*schema.Schema{
+			"cen_ids": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				ForceNew: true,
+				MinItems: 1,
+				MaxItems: 5,
```

Comment:
Please use `ids` instead '`cen_ids`, and I think it should not be limited in 1-5.